litejpeg/core/rle/entropycoder.py

```diff
@@ -0,0 +1,223 @@
+"""
+EntropyCoder Module:
+----------------
+This module is been made in order to calculate the number of bits to
+store the amplitude used in the huffman encoding.
+"""
+
+# Importing libraries.
+from litex.gen import *
+from litex.soc.interconnect.stream import *
+
+from litejpeg.core.common import *
+
+
+# To provide delay so in order to sink the data coming from the main
+# module to that of the Datapath module.
+datapath_latency = 3
+
+
+@CEInserter()
+class EntropyDatapath(Module):
+    """
+    EntropyDatapath :
+    ------------------
+    It contains the steps for the EntropyCoder to calculate the bits for the
+    Amplitude to store.
+
+    Attributes:
+    -----------
+    sink : Get the input from the Entrophycoder.
```

Comment:
Two things here;
 * Capitalization  should always match. If the class is called `EntrophyCoder` you should always refer to it as `EntrophyCoder` never `entrophyCoder` or `Entrophycoder`.
 * "Get the input from the Entrophycoder." is not informative. It should be something like "Accepts data which is encoded in huffman entropy foo bad, such as the output from the EntrophyCoder class." -- The difference is that it tells me *what* the data is.

Reply:
DONE